HHS/Head-Start-TTADP
# [TTAHUB-2205] Merge goals backend

## Description of change

Merge goals back end, as well as finishing touches on front end

## How to test

Test that you can merge goals, that you are redirected and shown a success message with the merged goals at the top. Please also take a look at the back end script and confirm that the data combinations the merge script performs makes sense

Excluded from this, in #1814 I am parsing the similarity API in order to exclude goals meeting certain conditions and combine them as in the RTR at large. If you see weirdness like what @AdamAdHocTeam describes below, please check to see if you are testing with goals that will be excluded (specifically to avoid weirdness)

**Goals excluded for MVP**: closed curated goals & goals that have no simple linkage back to an older grant

## Issue(s)

* https://ocio-jira.acf.hhs.gov/browse/TTAHUB-2205


## Checklists

### Every PR



### Production Deploy


### After merge/deploy
